src/main/kotlin/com/linchpino/core/service/InterviewService.kt

```diff
@@ -32,30 +34,42 @@ class InterviewService(
     private val jobPositionRepository: JobPositionRepository,
     private val interviewTypeRepository: InterviewTypeRepository,
     private val mentorTimeSlotRepository: MentorTimeSlotRepository,
-    private val accountService: AccountService
+    private val accountService: AccountService,
+    private val emailService: EmailService,
 ) {
 
     fun createInterview(request: CreateInterviewRequest): CreateInterviewResult {
         val jobSeekerAccount = accountRepository.findByEmailIgnoreCase(request.jobSeekerEmail)
-            ?: accountRepository.findReferenceById(accountService.createAccount(
-                CreateAccountRequest(
-                    firstName = null,
-                    lastName = null,
-                    email = request.jobSeekerEmail,
-                    password = null,
-                    status = AccountStatusEnum.DEACTIVATED,
-                    type = AccountTypeEnum.JOB_SEEKER.value
-                )
-            ).id)
+            ?: accountRepository.findReferenceById(
+                accountService.createAccount(
+                    CreateAccountRequest(
+                        firstName = "",
+                        lastName = "",
+                        email = request.jobSeekerEmail,
+                        password = null,
+                        status = AccountStatusEnum.DEACTIVATED,
+                        type = AccountTypeEnum.JOB_SEEKER.value
```

Comment:
the status and type fields are misordered as mentioned in the accountService.createAccount method